test/Test/Core/Issue.hs

```diff
@@ -0,0 +1,69 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Test.Core.Issue where
+
+import IW.App (AppEnv)
+import IW.Core.Id (Id (..))
+import IW.Core.Issue (Issue (..))
+import IW.Core.Repo (RepoName (..), RepoOwner (..))
+import IW.Core.SqlArray (SqlArray (..))
+import IW.Effects.Log (runAppLogIO)
+import IW.Db.Functions (asSingleRow, query)
+
+import Database.PostgreSQL.Simple.Types ((:.) (..))
+
+import Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+
+issueRoundtripProp :: AppEnv -> Property
+issueRoundtripProp env = property $ do
+    generatedIssue <- forAll genIssue
+    parsedIssue <- liftIO 
+                $ runAppLogIO env  
+                $ asSingleRow
+                $ query [sql| SELECT ?, ?, ?, ?, ?, ?, ? |] (Only (issueId generatedIssue) :. generatedIssue)
```

Comment:
This line is really long. I propose to move it into separate function like:

```haskell
issueViaSql :: WithDb m => Issue -> m Issue
issueViaSql issue = asSingleRow $ query
    [sql| SELECT ?, ?, ?, ?, ?, ?, ? |]
    (Only (issueId generatedIssue) :. generatedIssue)
```

After that it should be possible to write this in a single line:

```haskell
    parsedIssue <- liftIO $ runAppLogIO env $ issueViaSql issue
```